Assets/Scripts/Importing/FontsImporter.cs

```diff
@@ -0,0 +1,345 @@
+﻿using SanAndreasUnity.Importing.Conversion;
+using System.Collections.Generic;
+using UnityEditor;
```

Comment:
remove this using - `UnityEditor` namespace is not available in a built application